docs/support/tutorials/hpc-quick.md

```diff
@@ -0,0 +1,187 @@
+[submit your computations as batch jobs](../../computing/running/getting-started.md)
+to the SLURM job scheduler, which runs them as soon as the requested resources
+are available. In practice, this means that getting work done does not
+necessitate being tied to your workstation. While setting up this automation
+can require some more planning on your part, in the long run it makes it more
+reproducible both to yourself and other parties, such as reviewers and collaborators.
+
+You can access the command-line interface either by
+using the [shell applications](../../computing/webinterface/shell.md)
+featured in the web interface (Windows, Linux or macOS) or by
+[using an SSH client on your own workstation](../../computing/connecting.md)
+(Linux or macOS).
+
+## Available resources
+
+### Scientific software
+
+Even if your computations are not highly parallelizable, you can benefit from
+running them on a CSC supercomputer by making use of the variety of useful
+[scientific computing software](../../apps/index.md) that is available.
+Puhti is especially distinguished in this regard, having over a hundred
+pre-installed programs.
+
+CSC systems use [environment modules](../../computing/modules.md) for
+managing software environments. These modules cover everything from
+[compilers](../../computing/installing.md#compiling) to
+[programming languages](../../apps/by_discipline.md#mathematics-and-statistics)
+to workflow utilities like
+[Nextflow](../../apps/nextflow.md) and [Snakemake](../../apps/snakemake.md).
+Effectively running most of the installed software requires using the
+[command-line interface](#command-line-interface), so it is extremely useful to
+have a solid grasp of the
+[basics of the Linux operating system](./env-guide/index.md).
+
+While the pre-installed software covers a wide variety of use cases, it is not
+uncommon to need to
+[carry out installations of your own](../../computing/installing.md).
+Luckily, we have various
+compilers,
+[high-performance libraries](../../computing/hpc-libraries.md) and
+other utilities to facilitate this.
+
+You may also wish to develop your own scripts instead of using existing
+software. It is most efficient to start writing and testing your code on your
+own device, since running it on a shared system (which supercomputers
+invariably are) inevitably introduces some overhead. You should only start
+running your scripts on a supercomputer once you are ready for testing them on a
+larger scale or using specific resources like GPUs.
+
+!!! note "Checking for availability"
+    If you have a piece of scientific software in mind, it is quite probable
+    that we have it installed on Puhti. You can search for software using the
+    command `module spider <module-name>`. Most often the name of the software
+    module is simply the name of the software, and even if your search query
+    does not match the module name exactly, the search is case-insensitive and
+    supports partial matches.
```

Comment:
```suggestion
A variety of useful [scientific computing software](../../apps/index.md) are
available on CSC supercomputers.
Puhti is especially distinguished in this regard, having over a hundred
pre-installed programs. Our application pages include batch job script examples
and guidelines for running the pre-installed software efficiently on CSC
supercomputers. We highly recommend using them as a starting point!

CSC supercomputers use [environment modules](../../computing/modules.md) for
managing software environments. These modules cover everything from
[compilers](../../computing/installing.md#compiling) to
[programming languages](../../apps/by_discipline.md#mathematics-and-statistics)
and workflow utilities like
[Nextflow](../../apps/nextflow.md) and [Snakemake](../../apps/snakemake.md).
Running most of the installed software efficiently requires using the
[command-line interface](#command-line-interface), so it is extremely useful to
have a working knowledge of the
[basics of the Linux operating system](./env-guide/index.md).

While the pre-installed software cover a wide variety of use cases, it is also
possible to install your own applications on CSC supercomputers. The process
often differs from carrying out installations on your own computer, so make
sure to familiarize yourself with
[our installation instructions](../../computing/installing.md). For compiling
HPC applications we have various [compilers](../../computing/installing.md#compiling),
[high-performance libraries](../../computing/hpc-libraries.md) and
other utilities available to facilitate this. Note that some installations, such
as complex Python environments, benefit from
[containerization](../../computing/containers/overview.md).

You may also wish to develop your own scripts and programs instead of using existing
software. It is most efficient to start writing and testing your code on your
own device, since running it on a shared system (which supercomputers
invariably are) inevitably introduces some overhead. You should only start
running your scripts on a supercomputer once you are ready for testing them on a
larger scale or using specific resources like GPUs.

!!! note "Checking for availability"
    If you have a piece of scientific software in mind, it is quite probable
    that we have it installed on Puhti. Besides browsing Docs CSC, you can search
    for software on the command-line using the command `module spider <module name>`.
    Most often the name of the software module is simply the name of the software,
    and even if your search query does not match the module name exactly, the
    search is case-insensitive and supports partial matches.
```